kvdb-rocksdb/examples/memtest.rs

```diff
@@ -0,0 +1,130 @@
+use ethereum_types::H256;
```

Comment:
What is our policy on license headers, do you need to include them in every single file?
I don't really care, just want it to be consistent.

Reply:
Yes, until further notice we should include them. I am 99% sure it's wholly unneeded, but until we know, they should be there.

@NikVolf can add pls?